src/main/filesystem.ts

```diff
@@ -70,6 +72,20 @@ export class KangarooFileSystem {
   keystoreInitialized = () => {
     return fs.existsSync(path.join(this.keystoreDir, 'lair-keystore-config.yaml'));
   };
+
+  readOrCreatePassword() {
+    const pwPath = path.join(this.appDataDir, '.pw');
+    if (!fs.existsSync(pwPath)) {
+      const pw = uuidv4();
```

Comment:
Maybe using `nanoid` here would be better since its more cryptograhically secure

Reply:
Why is nanoid more secure than a uuid? In general I think the point here is just that it cannot be guessed which a uuid should satisfy.

Reply:
I believe `nanoid` is using the crypto node package under the hood, not sure if `uuid` is, but that would ensure we are generating secure passwords, although I think this is really more of a nitpick to be honest, uuid is still fine, especially if it can be overridden

Reply:
I replaced it with nanoid now since nanoid is being used already elsewhere in the code anyway so we can get rid of the uuid dependency too. They seem to be equally secure though (at least according to what they claim) but nanoids are shorter for the same security level because they're using a larger alphabet...